apis/v1alpha1/instrumentation_webhook.go

```diff
@@ -195,6 +195,37 @@ func (r *Instrumentation) ValidateDelete() (admission.Warnings, error) {
 	return nil, nil
 }
 
+func parseJaegerRemoteSamplerArgument(argument string) error {
+	parts := strings.Split(argument, ",")
+
+	for _, part := range parts {
+		kv := strings.Split(part, "=")
+		if len(kv) != 2 {
+			return fmt.Errorf("invalid argument: %s", part)
+		}
+
+		switch kv[0] {
```

Comment:
Should we check that some parameters are provided (e.g. endpoint) or can the SDK default to something meaningful? 

The default values are not defined in https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/sdk-environment-variables.md#general-sdk-configuration